### PR TITLE
Go bash (for) the make scripts

### PR DESCRIPTION
**Pull request by andreas-kupries**
https://trello.com/c/eX0agPsP/189-clean-up-the-names-of-makefile-targets-and-scripts
Related PRs:
- https://github.com/SUSE/cf-ci/pull/4
- https://github.com/SUSE/uaa-fissile-release/pull/25

### CODE REVIEW

File: make/uaa-releases [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+. "${GIT_ROOT}/.envrc"
+. "${GIT_ROOT}/src/uaa-fissile-release/.envrc"
+
+# FISSILE_DOCKER_REGISTRY and FISSILE_DOCKER_ORGANIZATION have the
+# same defaults, and pass through also. FOSSIL_REPOSITORY we must not
+# pass, on pain of getting images with the same name for different
+# roles in SCF and UAA (mysql, uaa, role-packages).
+
+unset FOSSIL_REPOSITORY
```
Comment (mook-as) [inline make/uaa-releases]: `FOSSIL`?

Comment (andreas-kupries) [inline make/uaa-releases]: Ouch. `FISSILE_` ... Will fix.

Comment (andreas-kupries) [inline make/uaa-releases]: Done with commit fd03b3b40b5c1c0adbbf7fc6438761af7665e027

Comment (andreas-kupries) [inline make/uaa-releases]: This includes the bash changes.

**Comment by andreas-kupries** [comment]
If there is one it is best to grep for more. ;)

**Pull request merged**